letsencrypt/tests/crypto_util_test.py

```diff
@@ -212,6 +212,17 @@ def test_parse_no_sans(self):
         self.assertEqual(
             [], self._call(test_util.load_vector('csr-nosans.pem')))
 
+class PyOpenSslLoaderTest(unittest.TestCase):
```

Comment:
nit1: not enough whitespace above that line
nit2: `PyOpenSSLLoadTest`
nit3: missing docstring: `"""Tests for letsencrypt.crypto_util._pyopenssl_load"""`
nit4; missing whitespace between `class` and `def`

hey, you may want to run `pep8` before submitting stuff :)